src/main/java/edu/oregonstate/cs361/battleship/Main.java

```diff
@@ -322,15 +322,15 @@ private static String fireAtNew(Request req) {
                 //System.out.println("Player");
                 //System.out.println(model.getShipByID(CArray[i].getName()).getHealth());
                 model.getShipByID(CArray[i].getName()).setHealth(CArray[i].getHealth() - 1);
-                /*System.out.println(model.getShipByID(CArray[i].getName()).getHealth());
+                System.out.println(model.getShipByID(CArray[i].getName()).getHealth());
```

Comment:
Pretty sure your suppose to deny the request until he makes the change on his own and remakes pull request. But I can be wrong.